src/layers/core/point-density-hexagon-layer/point-density-hexagon-layer.js

```diff
@@ -48,13 +53,10 @@ function _needsReProjectPoints(oldProps, props) {
   return oldProps.radius !== props.radius;
 }
 
-function _getCountRange(hexagons) {
-  return [
-    Math.min.apply(null, hexagons.map(bin => bin.points.length)),
-    Math.max.apply(null, hexagons.map(bin => bin.points.length))
-  ];
+function _percentilChanged(oldProps, props) {
```

Comment:
> My thinking is any method that doesn't require the this context can be a static method that's outside the layer

Understood. That's a good general programming principle. 

It's your call of course, but to explain my thinking further: in this case I do feel that the props are associated with the layer instance, partly because `props` is really just a short hand for `this.props`, and partly because the logic of diffing props does not seem to fit the description of an independent function, it is really a part of the internal logic of the layer. 

That said, if you extracted the individual props from the props objects and passed those instead of the props object, it would be easier for me to convince myself that this is a utility method (but then it probably wouldn't be small and clean).